components/menu-picklist/__examples__/tooltip-list-item.jsx

```diff
@@ -10,7 +12,9 @@ const ListItemRenderer = (props) => (
 		align="bottom left"
 		content={`${props.label} tooltip on bottom left`}
 	>
-		<p className="slds-truncate">{props.label} (Hover for tooltip)</p>
+		<p className="slds-truncate" tabIndex="0">
```

Comment:
`jsx-a11y/no-noninteractive-tabindex` is triggering because it doesn't have an ARIA role such as `button`. `role="button"` should fix this.

Reply:
That's not allowed to assign `role` to non-interactive, showing following error.
![image](https://user-images.githubusercontent.com/34851451/55161627-00ab2c00-518c-11e9-87f6-2845d116793d.png)

Similar approach is also used in other parts also like `components/app-launcher/tiles.jsx`.